receiver/elasticsearchreceiver/metadata.yaml

```
@@ -1 +1,268 @@
 name: elasticsearchreceiver
+
+attributes:
+  elasticsearch.cluster.name:
+    description: The name of the elasticsearch cluster.
+  elasticsearch.node.name:
+    description: The name of the elasticsearch node.
+  cache_name:
+    description: The name of cache.
+    enum:
+    - fielddata
+    - query
+  fs_operation:
+    value: operation
+    description: The type of file store operation.
+    enum:
+    - read
+    - write
+  generation:
+    description: The generation on which garbage collection was performed.
+    enum:
+    - young
+    - old
+  memory_segment:
+    value: segment
```

This is ok, but I think could just be:
```suggestion
  segment:
```

---

Actually just removed this label and split into two separate metrics to match what is already out there for JVM metrics (the ones documented here: https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/jmx-metrics/docs/target-systems/jvm.md#client-request-metrics)